batch/batch/worker/worker.py

```diff
@@ -443,9 +442,12 @@ async def _pull_image(self):
                 # image.
                 # FIXME improve the performance of this with a
                 # per-user image cache.
-                auth = self._current_user_access_token()
                 await docker_call_retry(
-                    MAX_DOCKER_IMAGE_PULL_SECS, str(self), docker.images.pull, self.image_ref_str, auth=auth
+                    MAX_DOCKER_IMAGE_PULL_SECS,
+                    str(self),
+                    docker.images.pull,
+                    self.image_ref_str,
+                    auth=self._current_user_access_token,
```

Comment:
I think this doesn't match the new signature because it's not async. But could just make `_current_user_access_token` async